src/v2/Components/v2/ArtworkFilter/ArtworkFilters/index.tsx

```diff
@@ -18,7 +19,7 @@ export const ArtworkFilters: React.FC = () => {
       <WaysToBuyFilter />
       <GalleryFilter />
       <InstitutionFilter />
-      <SizeRangeFilter />
+      <SizeFilter useFilterContext={useArtworkFilterContext} />
```

Comment:
This is `expanded` always, whereas the previous size filter defaulted to not expanded (but could override that via a prop).

Minor, but possibly unintentional!

By expanded, I mean on load the filter will be opened:

![Screen Shot 2020-06-01 at 6 19 16 PM](https://user-images.githubusercontent.com/1457859/83460501-91d2fe00-a434-11ea-815e-a7c36b73df96.png)

Reply:
in the design it says it should be always expanded.